docs/api.md

```diff
@@ -3223,7 +3223,7 @@ The default flags that Chromium will be launched with.
   - `userDataDir` <[string]> Path to a [User Data Directory](https://chromium.googlesource.com/chromium/src/+/master/docs/user_data_dir.md).
   - `env` <[Object]> Specify environment variables that will be visible to the browser. Defaults to `process.env`.
   - `devtools` <[boolean]> Whether to auto-open a DevTools panel for each tab. If this option is `true`, the `headless` option will be set `false`.
-  - `pipe` <[boolean]> Connects to the browser over a pipe instead of a WebSocket. Defaults to `false`.
+  - `webSocket` <[boolean]> Connects to the browser over a WebSocket instead of a pipe. Defaults to `false`.
```

Comment:
I'd love the defaults to be the same across browsers so that there are no surprises. Other than that, works for me.

cc @JoelEinbinder

Reply:
Defaults to false everywhere now.